parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -181,18 +176,20 @@ pub mod pallet {
 					return Err(Error::<T>::Overflow.into())
 				}
 
-				<MessageQueue<T>>::append(Message {
+				<MessageQueue<T>>::try_append(Message {
 					target,
 					nonce: *nonce,
 					payload: payload.to_vec(),
-				});
-				Self::deposit_event(Event::MessageAccepted(*nonce));
-				Ok(())
+				})
+				.map_err(|_unit| Error::<T>::QueueSizeLimitReached.into())
```

Comment:
We use this convention for unused params:
```suggestion
				.map_err(|_| Error::<T>::QueueSizeLimitReached.into())
```